api/src/main/java/mod/chiselsandbits/api/util/ReflectionUtils.java

```diff
@@ -36,4 +37,39 @@ public static Object getField(final Object target, final String name)
             throw new IllegalStateException("Failed to get value!");
         }
     }
+
+    public static Object getFieldInherited(final Object target, final String name)
```

Comment:
Please do not use reflection when accessing methods or fields or classes that can trivially be access transformed and access widened.

Especially since 1.20.1 is not running in unobfuscated mode, nor is fabric running in mojmap mode so this won't work.